### PR TITLE
Add Privacy policies for SparkRewards

### PR DESCRIPTION
**Pull request by mativanarquero**

### CODE REVIEW

**Comment by harveyjavier** [comment]
checking

File: sparkrewards/index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    </p>
+    <p><strong>Changes to This Privacy Policy</strong></p>
+    <p>
+      We may update our Privacy Policy from
+      time to time. Thus, you are advised to review this page
+      periodically for any changes. We will
+      notify you of any changes by posting the new Privacy Policy on
+      this page.
+    </p>
+    <p>This policy is effective as of 2021-03-25</p>
```
Comment (harveyjavier) [inline sparkrewards/index.html:121]: Please put the exact date of the release of SparkRewards

**Review by harveyjavier — CHANGES_REQUESTED** [review]

File: sparkrewards/index.html (hunk 1 above)
Comment (mativanarquero) [reply sparkrewards/index.html:121]: Copy @harveyjavier, thank you. Updating...

**Review by harveyjavier — APPROVED** [review]
Rewarding!

**Pull request merged**